nf_core/__main__.py

```diff
@@ -488,6 +447,151 @@ def lint(ctx, dir, release, fix, key, show_passed, fail_ignored, fail_warned, ma
         sys.exit(1)
 
 
+# nf-core pipelines subcommands
+@nf_core_cli.group()
+@click.pass_context
+def pipelines(ctx):
+    """
+    Commands to manage nf-core pipelines.
+    """
+    # ensure that ctx.obj exists and is a dict (in case `cli()` is called
+    # by means other than the `if` block below)
+    ctx.ensure_object(dict)
+
+
+# nf-core pipelines create
+@pipelines.command("create")
+@click.pass_context
+@click.option(
+    "-n",
+    "--name",
+    type=str,
+    help="The name of your new pipeline",
+)
+@click.option("-d", "--description", type=str, help="A short description of your pipeline")
+@click.option("-a", "--author", type=str, help="Name of the main author(s)")
+@click.option("--version", type=str, default="1.0dev", help="The initial version number to use")
```

Comment:
```suggestion
@click.option("--version", type=str, default="1.0.0dev", help="The initial version number to use")
```
we should encourage correct semver 🙂